nifi-mock/src/main/java/org/apache/nifi/util/MockProcessSession.java

```diff
@@ -753,23 +763,8 @@ public void transfer(final Collection<FlowFile> flowFiles, final Relationship re
             transfer(flowFiles);
             return;
         }
-        if(!processor.getRelationships().contains(relationship)){
```

Comment:
I replaced those by calling this [transfer(flowfile, relationship)](https://github.com/ijokarumawak/nifi/blob/6c15af2d89289d53fd51cdfe8e675b55bfe128f8/nifi-mock/src/main/java/org/apache/nifi/util/MockProcessSession.java#L743) that does the same validations.